add python version in README.md (#39)
Netuitive/docker-agent: README.md

Netuitive Agent Docker Container
=================================

This project creates a Docker container bundled with the [Netuitive Linux Agent](https://github.com/Netuitive/omnibus-netuitive-agent) and default Docker monitoring support.  It will automatically monitor your Docker host and containers.

For more information on the Netuitive Linux Agent, go [here](https://help.netuitive.com/Content/Misc/Datasources/Netuitive/new_netuitive_datasource.htm). For more information on the Netuitive Agent Docker Container, go [here](https://help.netuitive.com/Content/Misc/Datasources/Netuitive/integrations/new_netuitive_datasource_via_docker.htm). For additional help, contact Netuitive support at [support@netuitive.com](mailto:support@netuitive.com).

> Note that in this repository whenever you see python it will be assumed that it is Python 2.7.

Reviewer: Capitalized `python` and made the sentence less passive.

```suggestion
Note that all references in this repository to Python are for Python 2.7.
```
